site/docs/getting-started.md

```diff
@@ -72,6 +73,14 @@ INSERT INTO local.db.table VALUES (1, 'a'), (2, 'b'), (3, 'c');
 INSERT INTO local.db.table SELECT id, data FROM source WHERE length(data) = 1;
 ```
 
+Iceberg also adds row-level SQL updates to Spark, [`MERGE INTO`](./spark.md#merge-into) and [`DELETE FROM`](./spark.md#delete-from):
+
+```sql
+MERGE INTO local.db.target t USING (SELECT * FROM updates) u ON t.id = u.id
+WHEN MATCHED THEN SET t.count = t.count + u.count
```

Comment:
according to https://github.com/apache/iceberg/issues/2175, can you update to "UPDATE SET"?

Reply:
Updated and deployed. Thanks for catching this!